tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -1,28 +1,75 @@
 #!/usr/bin/env bash
+docker_script_dir=$(realpath $(dirname "$0"))
 
 # Default values
 CONFIG_FILE_IS_NEW=0
 export PYTHON_EGG_CACHE=/tmp
 set -e
 
+YELLOW='\033[1;33m'
+BROWN='\033[0;33m'
+GREEN='\033[1;32m'
+RED='\033[1;31m'
+NC='\033[0m' # No Color
+
+TRACIM_USER='www-data'
+
+function log {
+    if [ "$DEBUG" = "1" ]; then
+        echo -e "\n${YELLOW}[$(date +'%H:%M:%S')]${BROWN} $ $1${NC}"
+    fi
+}
+
+function loggood {
+    if [ "$DEBUG" = "1" ]; then
+        echo -e "\n${YELLOW}[$(date +'%H:%M:%S')]${GREEN} $ $1${NC}"
+    fi
+}
+
+function logerror {
+    echo -e "\n${RED}[$(date +'%H:%M:%S')]${RED} $ $1${NC}"
+    exit 1
+}
+
+function as_user {
+    su $TRACIM_USER -s "/bin/bash" -c "$1"
+}
+function tracimcli_as_user {
+    if [ "$DEBUG" = "1" ]; then
+      debug_string='-d'
+    else
+      debug_string=''
+    fi
+    as_user "tracimcli $1 $debug_string -c /etc/tracim/development.ini"
+}
+function alembic_as_user {
+    as_user "alembic -c /etc/tracim/development.ini $1"
+}
+
+
+
 # Check environment variables
-/bin/bash /tracim/tools_docker/Debian_Uwsgi/check_env_vars.sh
+log "Checking of docker env var"
+/bin/bash "$docker_script_dir/check_env_vars.sh"
 if [ ! "$?" = 0 ]; then
+    logerror "invalid env var"
     exit 1
 fi
+loggood "check of docker env var success"
 
 # Execute common tasks
-/bin/bash /tracim/tools_docker/Debian_Uwsgi/common.sh
+/bin/bash "$docker_script_dir/common.sh"
 if [ ! "$?" = 0 ]; then
     exit 1
 fi
 
 # Create file with all docker variable about TRACIM parameter
 printenv |grep TRACIM > /var/tracim/data/tracim_env_variables || true
-
 # Add variable for using xvfb with uwsgi
 echo "DISPLAY=:99.0" >> /var/tracim/data/tracim_env_variables
+chown www-data:www-data -R /var/tracim
```

Comment:
@grignards Can we discuss about that ? I need to explain what we really need and it is a good opportunity to change this (if it is possible).

Reply:
Add a log explaining what we do + the fact it can take time.